ci-operator/jobs/openshift/installer/openshift-installer-master-presubmits.yaml

```diff
@@ -1808,7 +1810,7 @@ presubmits:
     name: pull-ci-openshift-installer-master-e2e-metal-ipi-ovn-ipv6
     optional: true
     rerun_command: /test e2e-metal-ipi-ovn-ipv6
-    run_if_changed: ^([^d]|d(d|o(d|cd))*([^do]|o([^cd]|c[^ds])))*(d(d|o(d|cd))*(oc?)?)?$
+    skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)|(^(data/.*|pkg/.*|images)/(aws|azure|baremetal|gcp|kubevirt|libvirt|openstack|ovirt|vsphere)/)|(^vendor/github.com/metal3-io/)
```

Comment:
I wouldn’t skip data/ changes — that includes RHCOS boot image changes which definitely affects us.

Reply:
Sorry my brain isn’t a good regex parser, I guess this would only skip `data/.*/<platform>`, so rhcos.json would still get tested

Reply:
Yep, that's correct.
I really wish this field were a list of globs instead of a regex.